transactor/src/test/scala/io/mediachain/transactor/CborSerializationSpec.scala

```diff
@@ -43,12 +55,59 @@ object CborSerializationSpec extends BaseSpec {
       meta = Map("created" -> CString("the past"))
     )
 
+    val entityUpdateCell = EntityUpdateCell(
+      entity = multihashRef("foo"),
+      chain = Some(multihashRef("bar")),
+      meta = Map("favoriteColor" -> CString("Blue! No, Yellow!!!"))
```

Comment:
that's a good call.  Do you think it's worth using scalacheck to generate these?